internal/core/config/validation/validator.go

```diff
@@ -28,14 +31,19 @@ func ValidateCustomConfig(conf interface{}) error {
 
 // ValidateStruct uses the `validate` struct tags to do standard validation
 func ValidateStruct(confStruct interface{}) error {
+	en := en.New()
+	uni := ut.New(en, en)
+	trans, _ := uni.GetTranslator("en")
+
 	validate := validator.New()
+	validate.RegisterTagNameFunc(utils.YAMLNameOfField)
+	_ = en_translations.RegisterDefaultTranslations(validate, trans)
```

Comment:
What is being ignored here?

Reply:
An error that will never occur with the standard english translation.

Reply:
Ah, sounds good :D